tests/background/uri-info-test.js

```diff
@@ -1,72 +1,113 @@
 import * as uriInfo from '../../src/background/uri-info';
 import { toResult } from '../promise-util';
 import settings from '../settings.json';
-import { unroll } from '../util';
 
-describe('UriInfo.query', function () {
-  var badgeURL = settings.apiUrl + '/badge';
+describe('UriInfo', () => {
+  const badgeURL = `${settings.apiUrl}/badge`;
+  let fetchStub;
 
-  beforeEach(function () {
-    sinon.stub(window, 'fetch').returns(
-      Promise.resolve(
-        new window.Response('{"total": 1}', {
-          status: 200,
-          headers: {},
-        })
-      )
+  beforeEach(() => {
+    fetchStub = sinon.stub(window, 'fetch').resolves(
+      new window.Response('{"total": 1}', {
```

Comment:
```suggestion
      new Response('{"total": 1}', {
```

I don't think the `window.` should be necessary.